chain-core/src/init/config.rs

```diff
@@ -72,12 +73,32 @@ impl SlashRatio {
         }
     }
 
+    pub fn get_proportional(
+        self,
+        failing_validators: usize,
+        total_validators: usize,
```

Comment:
I'm thinking whether this makes sense against the number of nodes if the voting power (determined by the bonded stake of validators) isn't equal among them.
Perhaps failing_validators_vote_power / total_vote_power?